### PR TITLE
[SUPPORT][LIVE-7336] move evm to own package

### PR DESCRIPTION
**Pull request by chabroA**
### 📝 Description

Move evm family in coin-evm package

### ❓ Context

- **Impacted projects**: `` 
- **Linked resource(s)**: [LIVE-7336](https://ledgerhq.atlassian.net/browse/LIVE-7336)

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [ ] **No breaking changes**  || logic moved to new package

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

[LIVE-7336]: https://ledgerhq.atlassian.net/browse/LIVE-7336?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/coin-evm/src/signOperation.ts [hunk 1]
```diff
@@ -0,0 +1,127 @@
+import { Observable } from "rxjs";
```
Comment (henrily-ledger) [inline libs/coin-evm/src/signOperation.ts:1]: For some reason this code isn't marked has rename but changed, I guess you do not have any change expect some import right ?

**Review by henrily-ledger — APPROVED** [review]

File: libs/coin-evm/src/signOperation.ts (hunk 1 above)
Comment (chabroA) [reply libs/coin-evm/src/signOperation.ts:1]: Indeed it should be moved, will take a look at that 👀

Comment (chabroA) [reply libs/coin-evm/src/signOperation.ts:1]: @henrily-ledger 
This is actually just a Github PR ui thing (bug or feature, I don't know).
I assume Github UI displays the "new file" icon above a certain threshold of change in a moved file 🤷‍♂️

But on git side (and Github history), the history is maintained.

When running `git log --follow -- libs/coin-evm/src/signOperation.ts`, all the commits are displayed, including the onces from Kevin when the file was still in LLC.

This is similar to what have been observed here: https://github.com/LedgerHQ/ledger-live/pull/3306#issuecomment-1535988667

**Review by valpinkman — APPROVED** [review]
Good from DX side. TS integration is good, project is correctly setup with the right configurations updated. 👌 

Just unfortunate about the few `@ts-expect-error` but it can be dealt later on.

Approved

**Comment by chabroA** [comment]
Bot passes -> https://github.com/LedgerHQ/ledger-live/actions/runs/5122762065

**Pull request merged**